ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,38 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_values_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("complex"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
`valid` should be able to return all the data types supported by the function even if it's only complex.